app/enterprise/2.1.x/brain-immunity/alerts.md

```diff
@@ -4,65 +4,69 @@ title: Immunity Alerts
 
 ### Introduction
 
-Immunity is working behind the scenes to monitor all the traffic that comes through Kong Enterprise. When it sees an anomaly, Immunity will send an alert to Kong Manager. The alert can be found on the Service Map and the Alerts page. Alerts are built to signal the health of your microservices system and help pinpoint which endpoints are struggling.
+Immunity monitors all traffic that flows through Kong Enterprise. When an anomaly is detected, Immunity sends an alert to Kong Manager and displays on the Alerts dashboard. Alerts are built to signal the health of your microservices system and help pinpoint which endpoints are struggling.  
 
+### Alerts Dashboard
+Use the Alerts Dashboard in Kong Manager to view and manage alerts. When an alert is generated, it is automatically added to the Alerts Dashboard. The dashboard gives a high-level overview of identified alerts, including severity level, event type, status, and details about the alert. Click an alert to drill down into more details to further investigate the issue.
 
-### Prerequisites
-* Kong Enterprise installed and configured
-* Kong Collector Plugin installed and configured
-* Kong Collector App installed and configured
+![Introduction to Kong Immunity Alerts](/assets/images/docs/ee/alerts_dashboard.png)
 
 
-For more information, see the [Kong Brain and Kong Immunity Installation and Configuration Guide](/enterprise/{{page.kong_version}}/brain-immunity/install-configure).
+### Types of Alerts
+Immunity evaluates your traffic every minute and creates an alert when it detects an anomalous event on either of two entity types: endpoint traffic or consumer traffic.
 
-
-### Immunity Generated Alerts
-
-Immunity evaluates your traffic every minute, and creates an alert when it detects an anomalous event on either of two entity types: endpoint traffic and consumer traffic.
-
-* `Endpoint alerts` are generated from traffic belonging to one specific endpoint, for example, `GET www.testendpoint/start`.
+* `Endpoint alerts` are generated from traffic belonging to one specific endpoint, for example, GET www.testendpoint/start.
```

Comment:
```suggestion
* `Endpoint alerts` are generated from traffic belonging to one specific endpoint, for example, `GET www.testendpoint/start`.
```

Reply:
@tlblessing I'll go back and add the kong enterprise variable. I selected line wrapping (numerous times), but it's not working. Thx for the review.